columnphysics/icepack_atmo.F90

```diff
@@ -526,9 +525,7 @@ subroutine neutral_drag_coeffs (apnd,     hpnd,     &
       use icepack_tracers, only: tr_pond
 
       real (kind=dbl_kind), dimension (:), intent(in) :: &
-         apnd     ,& ! melt pond fraction of sea ice
-         hpnd     ,& ! mean melt pond depth over sea ice
-         ipnd     ,& ! mean ice pond depth over sea ice in cat n
```

Comment:
This isn't going to be backward compatible.  For now, I recommend just adding a comment that they aren't used.

Reply:
I take that back - this routine is internal to Icepack so it should be fine.  We only worry about backward compatibility for the interface routines.

Reply:
Correct.